test/integration/component/test_dhcp_dns_offload.py

```diff
@@ -0,0 +1,2126 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+""" Tests for DHCP DNS offload feature
+"""
+from marvin.cloudstackTestCase import cloudstackTestCase, unittest
+from marvin.lib.utils import (random_gen, validateList, cleanup_resources, get_hypervisor_type)
+from marvin.lib.base import (Account,
+                             NIC,
+                             ServiceOffering,
+                             DiskOffering,
+                             VirtualMachine,
+                             Network,
+                             NetworkOffering,
+                             Configurations,
+                             Host,
+                             Template,
+                             SSHKeyPair,
+                             Router)
+from marvin.lib.common import (get_zone,
+                               get_template,
+                               get_domain
+                               )
+from nose.plugins.attrib import attr
+from marvin.codes import PASS,FAIL
+from random import randint
+import random,string,time,tempfile,os,re
+
+@unittest.skip("Skipping dhcp-dns-offload tests for now, since it requires external dhcp server")
```

Comment:
Need to comment this if DHCP Server is available in the shared network broadcast domain while running tests. Looks good to me.